resources/scripts/openapi-icon-tool/.idea/jarRepositories.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RemoteRepositoriesConfiguration">
+    <remote-repository>
+      <option name="id" value="central" />
+      <option name="name" value="Maven Central repository" />
+      <option name="url" value="https://repo1.maven.org/maven2" />
+    </remote-repository>
+    <remote-repository>
```

Comment:
Why do we need this repository? Do we pull something specific from JBoss?

Reply:
Actually we use `Gradle` right? Do we need this file at all?

Reply:
This files are auto added one. These are removed while removing `.idea` folder